test/e2e/lib/flows/login-flow.js

```diff
@@ -122,11 +121,6 @@ export default class LoginFlow {
 		const navbarComponent = await NavBarComponent.Expect( this.driver );
 		await navbarComponent.clickCreateNewPost( { siteURL: siteURL } );
 
-		if ( usingGutenberg ) {
```

Comment:
Why are these no longer needed?

Reply:
I moved them to `_postInit`, but I just realized this is not correct, because `_postInit` runs every time we do `GutenbergEditorComponent.Expect( driver );` which will close the sidebar in some tests where we don't want that. Revert thins change in a bit.